addons/binding/org.openhab.binding.russound/src/main/java/org/openhab/binding/russound/internal/rio/RioConstants.java

```diff
@@ -19,118 +19,134 @@
 public class RioConstants {
 
     // BRIDGE TYPE IDS
-    public static final ThingTypeUID BRIDGE_TYPE_RIO = new ThingTypeUID(RussoundBindingConstants.BINDING_ID, "rio");
-    public static final ThingTypeUID BRIDGE_TYPE_CONTROLLER = new ThingTypeUID(RussoundBindingConstants.BINDING_ID,
+    public final static ThingTypeUID BRIDGE_TYPE_RIO = new ThingTypeUID(RussoundBindingConstants.BINDING_ID, "rio");
```

Comment:
Some `public final static` are still changed maybe you can do a replace-all 'final static' -> 'static final'